db/commitlog/commitlog.cc

```diff
@@ -258,8 +259,8 @@ class db::commitlog::segment_manager : public ::enable_shared_from_this<segment_
         uint64_t segments_destroyed = 0;
         uint64_t pending_flushes = 0;
         uint64_t flush_limit_exceeded = 0;
-        uint64_t total_size = 0;
         uint64_t buffer_list_bytes = 0;
+        uint64_t active_size_on_disk = 0;
         uint64_t total_size_on_disk = 0;
```

Comment:
So what is the difference between total and active? Can we have a comment describing which is which? may be rename them to be more clear?

Reply:
active == used by commitlog data, total == allocated (including unused, reserve and recycled)
but yes, needs commenting. And maybe better names in fact.